File: Programação Orientada a Objetos/Códigos/Códigos/POO 2/Projeto Pessoas/PessoaJuridica.java
package Proj_Pessoas;

public class PessoaJuridica extends Pessoa{
	private String cnpj;
	public String endereco;
	
	public PessoaJuridica() {
		
	}
	
	public String getCnpj() {
		return cnpj;
	}

	public void setCnpj(String cnpj) {
		this.cnpj = cnpj;
	}

	public String getEndereco() {
		return endereco;
	}

	public void setEndereco(String endereco) {
		this.endereco = endereco;
	}
}

File: Programação Orientada a Objetos/Códigos/Códigos/POO 4/Pessoas/PessoaJuridica.php
<?php
require_once 'Pessoa.php';
class PessoaJuridica extends Pessoa {
	//Atributos...
	private $cnpj;
	private $endereco;

	public function getCNPJ(){
		return $this->cnpj;
	}

	public function setCNPJ($cnpj): void{
		$this->cnpj = $cnpj;
	}

	public function getEndeco(){
		return $this->endereco;
	}

	public function setEndereco($endereco): void{
		$this->endereco = $endereco;
	}

}
?>